module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,19 @@
+#include "sort.hpp"
+
+bool isFirstLess(const std::string &first, const std::string &second) {
+    if (first.length() < second.size())
+        return true;
+    else if (first.length() == second.size()){
+        for (auto i = 0; i< first.length(); ++i)
+            if (first[i] < second[i])
+                return true;
+            else if (first[i] > second[i])
+                return false;
+    }
+    return false;
```

Comment:
If  you must use nested if-statements there is something wrong and you can make it simpler :)